pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentSubscription.java

```diff
@@ -1088,6 +1088,13 @@ public SubscriptionStatsImpl getStats(Boolean getPreciseBacklog, boolean subscri
                     consumerStats.keyHashRanges = consumerKeyHashRanges.get(consumer).stream()
                             .map(Range::toString)
                             .collect(Collectors.toList());
+                    long sameNameConsumerCount = consumerKeyHashRanges.keySet()
+                            .stream()
+                            .filter(c -> c.consumerName().equals(consumer.consumerName()))
+                            .count();
```

Comment:
Can we get the consumer count without going through all the items of consumerKeyHashRanges? If we have many consumers under a topic, this might introduce CPU spike